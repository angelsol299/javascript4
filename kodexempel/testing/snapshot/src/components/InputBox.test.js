import React from 'react';
import renderer from 'react-test-renderer';
import InputBox from './InputBox';

test('Rendering an input box', () => {
  const component = renderer.create(<InputBox />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
