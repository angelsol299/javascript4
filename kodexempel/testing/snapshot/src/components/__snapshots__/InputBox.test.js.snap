// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering an input box 1`] = `
Array [
  <h1>
    Hello
  </h1>,
  <input
    onChange={[Function]}
    type="text"
    value=""
  />,
]
`;
